src/utils/StorybookWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Global, css } from '@emotion/core';
+import { fontFaces, cssResets, previewStyles } from './renderHtmlDocument';
+
+type Props = {
+    children: React.ReactElement;
+};
+
+export const StorybookWrapper: React.FC<Props> = ({ children }: Props) => (
+    <div>
+        <Global
```

Comment:
Minor, but what about just using a style tag here?

    <style>`${fontFaces}${cssResets}${previewStyles`</style>

It is just as concise but doesn't require the `Global`, whose behaviour is unclear (I think it's more relevant to client-side loading but not sure?)

Reply:
Can do. I just thought the `Global` would just keep things a bit more _Reacty_ but I suppose we can live with a `style` tag instead. The only difference, AFAIK, is the styles will all be added to the <head> for our app and somewhere in the <body> for SB.